lab_5_scrapper/scrapper.py

```diff
@@ -13,70 +69,134 @@ def __init__(self, path_to_config: Path) -> None:
         """
         Initializes an instance of the Config class
         """
-        pass
+        self.path_to_config = path_to_config
+        self._validate_config_content()
+        self._config_dto = self._extract_config_content()
+        self._seed_urls = self._config_dto.seed_urls
+        self._num_articles = self._config_dto.total_articles
+        self._headers = self._config_dto.headers
+        self._encoding = self._config_dto.encoding
+        self._timeout = self._config_dto.timeout
+        self._should_verify_certificate = self._config_dto.should_verify_certificate
+        self._headless_mode = self._config_dto.headless_mode
 
     def _extract_config_content(self) -> ConfigDTO:
         """
         Returns config values
         """
-        pass
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            info = json.load(file)
+
+        config_dto = ConfigDTO(info['seed_urls'],
+                               info['total_articles_to_find_and_parse'],
+                               info['headers'],
+                               info['encoding'],
+                               info['timeout'],
+                               info['should_verify_certificate'],
+                               info['headless_mode'])
+
+        return config_dto
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters
         are not corrupt
         """
-        pass
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
```

Comment:
use `_extract_config_content` or refer to the corresponding `Config` attirbute instead of opening and reading the file again